unlock-app/src/components/interface/checkout/main/Select.tsx

```diff
@@ -307,7 +316,11 @@ export function Select({ checkoutService }: Props) {
     (lock?.isSoldOut && !(membership?.member || membership?.expired)) ||
     isNotExpectedAddress ||
     isLoadingHook ||
-    isSigning
+    isSigning ||
+    // Prevent isLoadingWaas from being null
+    !isLoadingWaas
+      ? undefined
+      : isLoadingWaas
```

Comment:
should it be a boolean? Then just use `!!isLoadingWas`. it will be `true` or `false` but never `null`.